lib/migrate/noEmitOnErrorsPlugin/noEmitOnErrorsPlugin.js

```diff
@@ -0,0 +1,55 @@
+const {
+	findPluginsByName,
+	safeTraverse
+} = require("../../utils/ast-utils");
+
+/**
+ *
+ * Transform for NoEmitOnErrorsPlugin. If found, removes the
+ * plugin and sets optimizations.noEmitOnErrors to true
+ *
+ * @param {Object} j - jscodeshift top-level import
+ * @param {Node} ast - jscodeshift ast to transform
+ * @returns {Node} ast - jscodeshift ast
+ */
+module.exports = function(j, ast) {
+	let rootPath;
+
+	// Remove old plugin
+	findPluginsByName(j, ast, ["webpack.NoEmitOnErrorsPlugin"])
```

Comment:
I noticed that this function is basically used also in the other PRs. I think it's worth creating on single function that does this. Regarding the `rootPath`, we could change the logic:
```js
const removed = findAndRemovePlugin(j, ast, ["webpack.NoEmitOnErrorsPlugin"]);
// if removed, means that was found
if (removed) {
  const rootPath = safeTraverse(path, ["parent", "parent", "parent", "value"]) // need to change the depth of the path obviusly
}
```

Ideally you could create on single PR to tackle all the transformers in one go so they can use the same utility all together

Reply:
I'll combine them all into this PR if that works for you. Seems like the easiest approach. On it now, thanks for the review.